docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -26,6 +29,9 @@ _usage(){
     echo -e "\n COMMANDS\n"\
             "   start\t\tstart miniziti (default)\n"\
             "   delete\t\tdelete miniziti\n"\
+            "   login\t\trun ziti edge login with miniziti context\n"\
+            "   show-admin\t\tprints admin user updb credentials\n"\
```

Comment:
If you think that we'll want to expand on the list of potential objects for the verb `show`, then we should go ahead and define the API like `miniziti verb object`, e.g., `show admin` rather than separate couplets for each permutation.

On the other hand, if it's truly a one-off, then introducing a single couplet `show-admin` might be the only thing that makes sense, as written, for the sake of simplicity in the implementation.

Reply:
Do you forsee any other show verbs at this time?

Reply:
I don't.

Reply:
Would `admin-creds` be better? Some alternative that avoids this pitfall?

Reply:
I think it's fine.